LWOTC_AlienPack/Src/LWOTC_AlienPack/Classes/UIScreenListener_Shell.uc

```diff
@@ -7,146 +7,146 @@
 
 class UIScreenListener_Shell extends UIScreenListener config(LWOTC_AlienPack);
 
-`include(LWOTC_AlienPack\Src\LWOTC_AlienPack.uci)
```

Comment:
Does this content need to be commented out or can it be added back in. Maybe add a task for this.